test/test.js

```diff
@@ -292,16 +293,57 @@ describe('onclickoutside hoc', function() {
   });
 
   describe('with child rendering as null', function() {
-    var StatelessComponent = () => null;
```

Comment:
Writing from mobile so i cant rly check the actual code now, but how would u like me to keep old test? Wasnt it testing an incorrect behaviour (now)? At least from whati remember. Wouldnt it just fail if unchanged?